packages/types/src/config-schema.json

```diff
@@ -2078,6 +2078,22 @@
           },
           "additionalItems": false,
           "description": "Mock configurations"
+        },
+        "initializeStore": {
+          "anyOf": [
+            {
+              "type": "object",
+              "additionalProperties": true
+            },
+            {
+              "type": "string"
+            },
+            {
+              "type": "array",
+              "additionalItems": true
+            }
+          ],
+          "description": "The path to the code runs before the store is attached to the schema"
```

Comment:
What does it mean "The path to the code"? Should it mean that this is executed before the store is attached to the schema?

Reply:
I explained it below :)